hid-xpadneo/src/hid-xpadneo.c

```diff
@@ -217,6 +217,36 @@ static const struct usage_map xpadneo_usage_maps[] = {
 
 	/* disable duplicate button */
 	USAGE_IGN(0xC0224),
+
+	/* XBE2 hacks */
+	/* Disable "dial", which is a redundant representation of the D-Pad */
+	USAGE_IGN(0x10037),
+
+	/* Disable unnecessary axes Vx, Vy, Vz, Vbrx, Vbry, Vbrz which cause an additional axis to show up */
```

Comment:
I thought it'd be semantically weird to only remove one of these, so I decided to just ignore all of them, but this _might_ actually be a bad idea - let me know what you think

Reply:
Are one of the paddles analog axes? Or are they just digital?

Reply:
This was about the last axis, lagging behind LX.

All of the paddles are buttons, which appear in `evtest` as `type 1 (EV_KEY), code 240 (KEY_UNKNOWN), value 0` (where the value represent which paddles are pressed)

Reply:
Okay, they should become `BTN_*` symbols then. I think it's okay to remove the VR axes then.

Reply:
Make the header comment shorter, like "disable blind axes" or something like that, then but the symbol name as a trailing comment on each line.